iceoryx2-bb/posix/tests/udp_socket_tests.rs

```diff
@@ -107,12 +107,12 @@ fn udp_socket_when_socket_goes_out_of_scope_address_is_free_again() {
 fn udp_socket_server_has_correct_address() {
     let sut_server = UdpServerBuilder::new()
         .address(ipv4_address::LOCALHOST)
-        .port(Port::new(65111))
+        .port(Port::new(55223))
```

Comment:
Nothing for this PR and I think you already did it somewhere else but using an atomic base port number an atomically increasing it for each test will make these tests more robust.